doc/user/content/integrations/llm.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Large Language Models
+description: "Learn how to integrate Materialize with Large Language Models (LLMs) using MCP"
+make_table_row_headers_searchable: true
+menu:
+  main:
+    parent: "integrations"
+    weight: 24
+---
+
+The Model Context Protocol (MCP) Server for Materialize lets large language models (LLMs) call your indexed views as real-time tools.
+The MCP Server automatically turns any indexed view with a comment into a callable, typed interface that LLMs can use to fetch structured, up-to-date answers—directly from the database.
+
+These tools behave like stable APIs.
+They're governed by your SQL privileges, kept fresh by Materialize's incremental view maintenance, and ready to power applications that rely on live context instead of static embeddings or unpredictable prompt chains.
+
+## Get Started
+
+We recommend using [uv](https://docs.astral.sh/uv/) to install and run the server.
+It provides fast, reliable Python environments with dependency resolution that matches pip.
+
+If you don't have uv installed, you can install it first:
+
+```bash
+$ curl -LsSf https://astral.sh/uv/install.sh | sh
```

Comment:
Oh ... just realized we should delete the prompt `$` -- the copy button copies it